packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -449,7 +449,6 @@ class Uploader {
       uploadUrl: this.options.uploadUrl,
       // @ts-ignore
       uploadLengthDeferred: false,
-      resume: true,
```

Comment:
The `resume` option did not have any effect when used inside Node.js without specifying the `urlStorage` parameter as well. So this can be removed.